introduction.md

```diff
@@ -1,2 +1,5 @@
 This is my introduction
 My favorite Code School Path is Ruby
+Deadly skills
+*Git
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?